searx/static/themes/oscar/src/js/01_init.js

```diff
@@ -1,20 +1,20 @@
 /**
- * searx is free software: you can redistribute it and/or modify
+ * SearXNG is free software: you can redistribute it and/or modify
  * it under the terms of the GNU Affero General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
  *
- * searx is distributed in the hope that it will be useful,
+ * SearXNG is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Affero General Public License for more details.
  *
  * You should have received a copy of the GNU Affero General Public License
- * along with searx. If not, see < http://www.gnu.org/licenses/ >.
+ * along with SearXNG. If not, see < http://www.gnu.org/licenses/ >.
  *
  * (C) 2019 by Alexandre Flament
  */
```

Comment:
Lets replace by .. SPDX

     # SPDX-License-Identifier: AGPL-3.0-or-later

Reply:
According to 
* https://opensource.google/docs/thirdparty/javascript/#spdx 
* https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/

it should be something like this:
```
/**
 * @license
 * (C) Copyright Contributors to the SearXNG project.
 * (C) Copyright Contributors to the searx project (2014 - 2021).
 * SPDX-License-Identifier: AGPL-3.0-or-later
 */
```

or for some files: 
```
/**
 * @license
 * (C) Copyright Contributors to the SearXNG project.
 * (C) Copyright Contributors to the searx project (2014 - 2021).
 * (C) 2014 by Thomas Pointhuber, <thomas.pointhuber@gmx.at>
 * SPDX-License-Identifier: AGPL-3.0-or-later
 */
```

Reply:
>> Lets replace by .. SPDX

> it should be something like this: ...

perfect!